just-pandas-things.ipynb

```diff
@@ -3360,6 +3360,308 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "---\n",
+    "## 7. Bugs\n",
```

Comment:
let's call this `Known bugs` and mention the links to the tracking PRs earlier in the post so people know that it's a well-known bug and not just a personal opinion.

Reply:
Done in #8